server/data/products.js

```diff
@@ -93,6 +94,7 @@ let exportedMethods = {
       logo: logo,
       tags: tags,
       developer: developer,
+      reviews: {},
```

Comment:
I think this should be an []

Reply:
I've replaced this and made it into an array.